test/unit/app.js

```diff
@@ -1277,7 +1270,7 @@ describe('Unit tests for swiftserver:app', function () {
           appType: 'scaffold',
           appName: applicationName,
           capabilities: [],
-          services: { objectstorage: {} }
+          services: { objectStorage: {} }
```

Comment:
doesn't actually test the key `objectStorage` exists unless we put something real in the value (not the empty object)